common/primitives/src/handles.rs

```diff
@@ -94,3 +94,21 @@ pub struct PresumptiveSuffixesResponse {
 	/// The suffixes
 	pub suffixes: Vec<HandleSuffix>,
 }
+
+/// Output response for retrieving the next suffixes for a given handle
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq)]
+pub struct CheckHandleResponse {
```

Comment:
I cannot think of anything else we might want to be returned, but if you can, let me know and I'll add it

Reply:
Any value in returning the actual suffix (or next N suffixes) instead of/in addition to the suffix index, to eliminate potentially having to make a separate call to the `get_next_available_suffixes` RPC?